### ISSUE TITLE
Visible Label Instead of Placeholder Text in Search Bar 

### ISSUE THREAD

**Issue by mtribone** [samvera/hyrax#3129]
### Descriptive summary

In Hyrax 2.1.0 RC3 where there is a search bar (home page, my works, my collections, etc.), the label for the input is hidden and the placeholder text disappears when a user starts to type. It is better for accessibility and usability to keep the label visible at all times.

### Rationale

When label elements are associated with input elements the label is spoken by screen readers when the field receives focus and users with impaired motor control are helped by a larger clickable area for the control, since clicking on the label or the control will activate the control.

Field labels located in close proximity to the associated field assist users of screen magnifiers because the field and label are more likely to visible within the magnified area of the page.

### Expected behavior

Form elements should have visible labels.

### Actual behavior

Label is hidden and placeholder text disappears when user begins to type.

### Related work

Violates [WCAG 3.3.2](https://www.w3.org/TR/UNDERSTANDING-WCAG20/minimize-error-cues.html#minimize-error-cues-intent-head) guidelines

**Comment by no-reply** [comment]
Thanks for this. What is the next step?

Do we need design/wireframes, and/or a catalog of places where this accessibility failure arises?

**Comment by mtribone** [comment]
Yeah a little design might be nice for this, so that people can see how the label is positioned with the search bar on various screen sizes.

The report from the firm indicates that just the placeholder text without visible label is in the dashboard/my/works and dashboard/my/collections areas of Hyrax, but it is probably anywhere the search input is being used; for example, homepage, catalog search. And it could be affecting any input label. The search was pinpointed in the report.

We don't use Hyrax at Penn State currently, so I'm not sure of all of the places this might be located. So yeah a list of all of the places where this search input or similarly styled input is being used, some wireframes, and deciding on the wording for the label would be good next steps.

**Comment by adamjarling** [comment]
@mtribone Is it acceptable to have the search input label be `sr-only`?  Or is that what you mean by the label being 'hidden'?

**Comment by adamjarling** [comment]
Currently there's a `sr-only` attribute on the `<label>`s, so if that works then can close this ticket.  If we'd prefer a label shown in all cases, I have that set up and can push it.  It'd look like this:

![home-search-label](https://user-images.githubusercontent.com/3020266/45439687-1d649400-b680-11e8-875d-4d3254da234b.png)

**Comment by mtribone** [comment]
@adamjarling Yeah having the label visible is what the firm recommends in their report. It makes the area more clickable for people and for people using magnifiers more readable. We've hidden it in the past, but it is more useable for more people if visible.

**Comment by mtribone** [comment]
I wonder if the US Design System hides the label in the example because it is preceded by an `<h6>`. Oh...they have skipped heading levels. oops.

**Comment by mtribone** [comment]
<img width="1129" alt="screen shot 2018-09-12 at 12 42 07 pm" src="https://user-images.githubusercontent.com/4163828/45440061-4fc6bf00-b689-11e8-977b-bcac3c5bbad9.png">
